packages/client/components/NewMeetingTeamPickerAvatars.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {useMemo} from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../styles/paletteV3'
+import defaultUserAvatar from '../styles/theme/images/avatar-user.svg'
+import {NewMeetingTeamPickerAvatars_team$key} from '../__generated__/NewMeetingTeamPickerAvatars_team.graphql'
+import Avatar from './Avatar/Avatar'
+import ErrorBoundary from './ErrorBoundary'
+
+const Container = styled('div')<{count: number}>(({count}) => ({
+  width: 44,
+  display: 'flex',
+  justifyContent: count > 1 ? 'flex-start' : 'center',
+  flexWrap: 'wrap',
+  bottom: count > 2 ? 4 : 0,
+  position: 'relative'
+}))
+
+const AvatarWrapper = styled('div')<{count: number}>(({count}) => ({
+  width: count > 1 ? 20 : 'auto',
+  height: count > 2 ? 20 : 'auto'
+}))
+
+const StyledAvatar = styled(Avatar)({
+  border: `2px solid ${PALETTE.SLATE_200}`
+})
+
+const getShuffledArr = (arr) => {
+  const newArr = arr.slice()
+  for (let i = newArr.length - 1; i > 0; i--) {
+    const rand = Math.floor(Math.random() * (i + 1))
+    ;[newArr[i], newArr[rand]] = [newArr[rand], newArr[i]]
+  }
+  return newArr
+}
+
+interface Props {
+  teamRef: NewMeetingTeamPickerAvatars_team$key
+}
+
+const NewMeetingTeamPickerAvatars = (props: Props) => {
+  const {teamRef} = props
+
+  const team = useFragment(
+    graphql`
+      fragment NewMeetingTeamPickerAvatars_team on Team {
+        teamMembers {
+          id
+          picture
+        }
+      }
+    `,
+    teamRef
+  )
+
+  const {teamMembers} = team
+
+  if (!teamMembers) {
```

Comment:
-1 It seams `teamMembers` is not nullable and it would violate the hook precondition when you return before `useMemo`.
If you're expecting this as some known exception in rare cases, please add a comment as explanation.

Reply:
Removed null check, not needed here